### ISSUE TITLE
Crash upon resetting save file and trying to start new save

### ISSUE THREAD

**Issue by SomeRandomInternetGuy9** [Creckeryop/3dsCookieClicker#2]
My o2ds crashed upon trying to reset my save after reaching Billionaire. Upon resetting my save and trying to start a new save, my o2ds popped up with a crash dump and restarted. 

This is an older game, but i would very much like some help here.

**Comment by Creckeryop** [comment]
Is your o2ds broken now, or its only game issue?

**Comment by SomeRandomInternetGuy9** [comment]
Only game issue, luckily. I don't think I can play Cookie Clicker anymore, though.

**Comment by Creckeryop** [comment]
i think you should delete ccsave.sav file (can't remember it's location) by yourself with FBI shell

**Comment by damiandoky37** [comment]
yeah but wouldnt be annoying to not have your score save and everytime you have to exit and delete a file?
